templates/base/resources/usr/local/bin/update_ubuntu.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo "Updating Ubuntu."
+apt-get update
+apt-get upgrade --yes
+apt-get clean
```

Comment:
@zsoltbalogh, why did you decide to have a separate script just to update Ubuntu? Can't you move this code to the script `attach_ubuntu_pro_subscription.sh`?

Reply:
Please check the ticket, you can see that we should still recommend updating Ubuntu and this is why the script is separated. It was initially together, but when I wrote the docs, I separated it.